src/common/entity/compute_state_display.ts

```diff
@@ -19,6 +19,9 @@ export const computeStateDisplay = (
   if (compareState === UNKNOWN || compareState === UNAVAILABLE) {
     return localize(`state.default.${compareState}`);
   }
+  if (compareState === "") {
+    return localize(`state.default.${UNAVAILABLE}`);
```

Comment:
this should be unknown.
```suggestion
    return localize(`state.default.${UNKNOWN}`);
```

Reply:
I was 100% sure i will pick the wrong option, whatever I choose :D